lib/CMakeLists.txt

```diff
@@ -51,6 +51,12 @@ set(SOURCES
 add_llvm_loadable_module(Svf ${SOURCES})
 add_llvm_Library(LLVMSvf ${SOURCES})
 
+# It seems that the Svf library needs to explictly require the CUDD alloc/init/etc so LLVMCudd
```

Comment:
I believe I have a fix but have been in VM purgatory today so but basically just used CMAKE_SYSTEM_NAME and a little if-else..   so just will test again, today got away from me but this is the easy part...